src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -769,6 +771,252 @@ protected bool IsAlive
             }
         }
 
+        /// <summary>
+        /// Gets the child Elements of the current Element.
+        /// </summary>
+        /// <returns>Child Elements.</returns>
+        public IEnumerable<Element> GetChildElements()
+        {
+            return GetElementChildElements(this.InternalElement);
+            
+        }
+
+        private IEnumerable<Element> GetElementChildElements(Autodesk.Revit.DB.Element element)
+        {
+            List<Element> components = new List<Element>();
+            BuiltInCategory builtInCategory = (BuiltInCategory)System.Enum.Parse(typeof(BuiltInCategory),
```

Comment:
what if this fails - is it possible the category is not a built in category?

Reply:
as far as I know it cant? @AndyDu1985 @ZiyunShang any thoughts on this?

Reply:
For element.Category, I think they should be builtInCategory. And user may add sub category to a category which may not be a builtInCategory, I'm not sure about it. 
I suggest to add a check to see if `builtInCategory` is `INVALID` .

Reply:
Any updates?